server/lib/report-manager.js

```diff
@@ -83,15 +83,16 @@ class ReportManager
 
     async getTextContent(accounts, operationsByAccount, frequency) {
 
-        let subject;
+        let freq;
```

Comment:
Alternatively, can you rename the `frequency` parameter to `frequencyKey` and `freq` to `frequency`?